index.html

```diff
@@ -347,9 +151,70 @@ <h3>Skills</h3>
       <img id="indicator" src="assests/images/shape.png" alt="Home Indicator/Light">
     </div>
   </footer>
-
+  
+    <!-- Popup Window -->
+    <div class="modal-overlay">
+      <div class="modal-container">
+          <div class="modal-header">
+            <div class="heading">
+              <span class="close-btn">&times;</span>
+              <p class="mobile-heading">Multi Post Stories</p>
+              <p class="desktop-heading">Keeping track of hundreds of components website</p>
+            </div>
+            <div class="modal-tags">
+              <ul>
+                  <li class="modal-tag">Html</li>
+                  <li class="modal-tag">Bootstrap</li>
+                  <li class="modal-tag">Ruby on Rails</li>
+              </ul>
+            </div>
+          </div>
+  
+          <div class="modal-body">
```

Comment:
- [ ] Kindly reduce the margins/ Paddings for the spacing that is between the top section of your popup and the popup body. This will ensure that your design matches the chosen Figma design.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/203615018-ffaf4b03-5c95-422d-8d25-203729136e60.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/203615093-37c4c236-3aa1-417c-9eec-eb454a1aac97.png)